launch/launch/actions/group_action.py

```diff
@@ -88,19 +88,20 @@ def parse(cls, entity: Entity, parser: Parser):
         _, kwargs = super().parse(entity, parser)
         scoped = entity.get_attr('scoped', data_type=bool, optional=True)
         forwarding = entity.get_attr('forwarding', data_type=bool, optional=True)
-        args = entity.get_attr('arg', data_type=List[Entity], optional=True)
+        keeps = entity.get_attr('keep', data_type=List[Entity], optional=True)
         if scoped is not None:
             kwargs['scoped'] = scoped
         if forwarding is not None:
             kwargs['forwarding'] = forwarding
-        if args is not None:
-            kwargs['launch_configurations'] = dict()
-            for e in args:
-                arg_name = tuple(parser.parse_substitution(e.get_attr('name')))
-                arg_value = parser.parse_substitution(e.get_attr('value'))
-                kwargs['launch_configurations'][arg_name] = arg_value
+        if keeps is not None:
+            kwargs['launch_configurations'] = {
+                    tuple(parser.parse_substitution(e.get_attr('name'))):
```

Comment:
@djchopp nit^(N!): I'd use `as` or instead of `name`. @wjwwood @ivanpauno sanity check this.

Reply:
@hidmic Should I put this in as well? I don't have strong feelings to which one should be used. I was waiting for input from @wjwwood and @ivanpauno but have not seen anything.

Reply:
No need to. If nobody else feels strongly about it, perhaps it's not worth the change.

Reply:
I don't mind either.